help-beta/README.md

```diff
@@ -0,0 +1 @@
+# Help Center Beta Docs
```

Comment:
This is temporary text, so this string doesn't matter, but this is probably a good moment to mention we have a "sentence case" policy for all strings, so this should be something like "Help center beta docs" or really "Help center beta").

Worth mentioning because that policy saves a lot of time in review, since it's easy for everyone to self-implement in their work. I guess we only mention it in
https://zulip.readthedocs.io/en/latest/translating/internationalization.html, but should maybe mention it elsewhere too.

Reply:
Fixed!